src/main/windows/windowManager.ts

```diff
@@ -371,7 +371,16 @@ export function switchServer(serverName: string) {
         nextTab = openTabs.find((e) => e.order === 0) || openTabs[0];
     }
     const tabViewName = getTabViewName(serverName, nextTab.name);
-    status.viewManager?.showByName(tabViewName);
+    if (waitForViewToExist) {
+        const timeout = setInterval(() => {
```

Comment:
Needed to wait for the view to be added to the view manager before we try to switch to it.